Algorithms/MaxPair.java

```diff
@@ -0,0 +1,68 @@
+package main;
+
+public class MaxPair {
+
+	public static class Pair
+	{
+		private int left;
+		private int right;
+			
+		public Pair(int left, int right) {
+			super();
+			this.left = left;
+			this.right = right;
+		}
+		
+		public int getLeft() {
+			return left;
+		}
+		public void setLeft(int left) {
+			this.left = left;
+		}
+		public int getRight() {
+			return right;
+		}
+		public void setRight(int right) {
+			this.right = right;
+		}
+		
+		public int getSum() {
+			return left + right;
+		}
+
+		@Override
+		public String toString() {
+			return "Pair [left=" + left + ", right=" + right + "]";
+		}
+	}
+	
+	public static Pair getMaxPair(Pair[] input) {
+		if (input == null || input.length < 1)

```

Comment:
What if the input array is empty ? Change input.length to !input.isEmpty()

Reply:
Well arrays don't have an `isEmpty` method, do you want me to use a List instead?

Reply:
@fallazc Yes please. Else input.length <=0 also would work